portal/src/components/v2/TextField/TextField.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import cn from "classnames";
+import { TextField as RadixTextField } from "@radix-ui/themes";
+import styles from "./TextField.module.css";
+
+type TextFieldSize = "2" | "3";
+
+function sizeToLabelClass(size: TextFieldSize) {
+  switch (size) {
+    case "2":
+      return styles["textField__label--size2"];
+    case "3":
+      return styles["textField__label--size3"];
+  }
+}
+
+export interface TextFieldProps {
+  darkMode?: boolean;
+  size: TextFieldSize;
+  label?: React.ReactNode;
+  disabled?: boolean;
+  readOnly?: boolean;
+  placeholder?: string;
+  error?: React.ReactNode;
+  iconStart?: React.ReactNode;
+  iconEnd?: React.ReactNode;
```

Comment:
For the same reason as `iconStart` of TextButton, can we hide `iconStart` and `iconEnd`? Or we can provide a dropdown to allow the storybook user to select a few common icons? Note that dropdown should have an option to specify `iconStart = undefined`.